### ISSUE TITLE
Mention relying color alone is an accessibility issue

### ISSUE THREAD

**Issue by takenspc** [ResponsiveImagesCG/ri-usecases#48]
In [3.6 Matching media features and media types](http://usecases.responsiveimages.org/#matching-media-features-and-media-types), current Use Case Requirement says:

> However, displaying a color image on monochrome media (e.g., paper and e-ink displays) can be problematic: different colors with similar luminosity are impossible to distinguish on monochrome media. This problem is illustrated in the figure below, where it becomes nearly impossible to associate slices of a pie chart with corresponding labels.
> 
> <mark>(snip)</mark>
> 
> The only feasible solution is for authors to provide alternative image content that communicates effectively in monochrome media. 

This is a common accessibility issue/mistake and fails WCAG 2.0 [Success Criterion 1.4.1 Use of Color](http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html).

Yet there are some cases where authors cannot modify existing problematic contents, Use Case Requirements should mention this is an accessibility issue and authors are encouraged to avoid relying color alone.

Related resources:
- [WebAIM  Web Accessibility for Designers](http://webaim.org/resources/designers/)
- [Good and Bad Examples of Use of Color](http://wearecolorblind.com/examples)

**Comment by nwtn** [comment]
+1. I know we talked about this, but I don't remember the outcome. I'll see if I can dig up the previous conversations in case there was anything already written.

**Comment by marcoscaceres** [comment]
@takenspc agree, can you suggest some text please to fix the section?

**Comment by nwtn** [comment]
[The IRC conversation I was thinking of](http://ircbot.responsiveimages.org/bot/log/respimg/2013-08-29#T31691). I believe it started out (in my mind) as "this needs to be a use case", and quickly got derailed into "the media queries don't work properly".

Also, this is one of @hellogeri's areas of expertise. Any comments/ideas?

**Comment by takenspc** [comment]
Here are some sentences (though I wonder these are appropriate for a spec document.)

> However, displaying a color image on monochrome media (e.g., paper and e-ink displays) can be problematic: different colors with similar luminosity are impossible to distinguish on monochrome media. This problem is illustrated in the figure below, where it becomes nearly impossible to associate slices of a pie chart with corresponding labels.
> <mark>(start here)</mark>
> This is a common accessibility issue ([WCAG 2.0 Success Criterion 1.4.1 Use of Color](http://www.w3.org/TR/WCAG20/#visual-audio-contrast-without-color)). Developers are highly encouraged to avoid relying color alone. There are some techniques to avoid it, such as connecting slices and labels with lines and using textures or patterns. For more details, the [Colour Accessibility](http://24ways.org/2012/colour-accessibility/) introduces backgrounds and some techniques of the color accessibility.
> <mark>(end here)</mark>

**Comment by marcoscaceres** [comment]
@takenspc thanks for the proposed text. The document is really just supposed to outline the use cases, not provide best practice guidelines for developers. how about instead:

> This is a well-known accessibility problem, as described in WCAG 2.0 (linked to the right section). A solution that affords developers the ability to discriminate on media type could potentially increase the accessibility of image content.

If you are ok with it, I will add it.

**Comment by marcoscaceres** [comment]
@takenspc we've merged in the proposed text. If you want us to modify it, just let us know.

**Comment by takenspc** [comment]
I confirmed. Thanks!
